refinery/data_set_manager/tests.py

```diff
@@ -2062,14 +2059,6 @@ def test_prepare_node_no_file_import_task_id_yet(self):
             expected_download_url=PENDING
         )
 
-    def test_prepare_node_no_file_store_item(self):
```

Comment:
Need to follow up on this unit test which was failing. Waiting for the fileStore pull request to merge also.